util/Version.h

```diff
@@ -1,15 +1,41 @@
 #ifndef _Version_h_
 #define _Version_h_
 
+/** @file
+ * @brief  Declares free functions to access the application and dependencies
+ *      versions.
+ */
+
 #include <string>
 #include <map>
 
 #include "Export.h"
 
+/** @brief  Returns the version string of FreeOrion.
+ *
+ * @return  The version string with the format
+ *      <em>VERSION BRANCH_NAME [build DATE.SHORT_COMMIT_HASH] BUILD_SYSTEM</em>
+ */
 FO_COMMON_API const std::string& FreeOrionVersionString();
 
+/** @brief  Returns a map of dependency versions.
+ *
+ * @return  A map with the dependency versions.  The key represents the
+ *      dependency name, the value corresponding dependency version string.
+ *
+ * @bug  The current implementation doesn't expose all dependencies of all
```

Comment:
Is it really a bug? Depends if a "limitation" is a bug... Or could say it just returns the dependency versions for common code.